resources/views/home/landing.blade.php

```diff
@@ -0,0 +1,169 @@
+                </span>
+                @endfor
+            </div>
+            <div class="landing-download">
+                <div class="landing-download__inner">
+                    <a href="http://m1.ppy.sh/r/osu!install.exe" class="landing-download__button shadow-hover">
+                        <span class="fa fa-cloud-download landing-download__icon"></span>
+                        <span class="landing-download__content-wrapper">
+                            <span class="landing-download__content landing-download__content--top">{{ trans("home.landing.download._") }}</span>
+                            <span class="landing-download__content landing-download__content--bottom js-download-platform"></span>
+                        </span>
+                    </a>
+                    <a href="{{ route('download') }}" class="landing-download__other js-download-other"></a>
+                </div>
+            </div>
+            <div class="js-landing-graph landing-graph">
+                <div class="landing-graph__info">
+                    <b>{{ number_format($totalUsers, 0) }}</b> {{ trans("home.landing.players") }}, <b>{{ number_format($currentOnline, 0) }}</b> {{ trans("home.landing.online") }}
+                </div>
+            </div>        
+        </div>
+    </header>
+    <main class="osu-layout__row osu-layout__row--landing">
+        <div class="landing-middle-buttons">
+            <div class="osu-layout__col-container">
+                <a href="#" class="osu-layout__col osu-layout__col--link osu-layout__col--sm-4 osu-layout__col--lg-4">
+                    <img class="landing-middle-buttons__image shadow-hover" src="/images/layout/landing-page/middle-button-1.jpg" alt="Placeholder text!">
+                </a>
+                <a href="{{ action('StoreController@getListing') }}" class="osu-layout__col osu-layout__col--link osu-layout__col--sm-4 osu-layout__col--lg-4">
+                    <img class="landing-middle-buttons__image shadow-hover" src="/images/layout/landing-page/middle-button-2.jpg" alt="osu!store">
+                </a>
+                <a href="//next.ppy.sh/" class="osu-layout__col osu-layout__col--link osu-layout__col--sm-4 osu-layout__col--lg-4">
+                    <img class="landing-middle-buttons__image shadow-hover" src="/images/layout/landing-page/middle-button-3.jpg" alt="osu!next">
+                </a>
+            </div>
+        </div>
+    </main>
+    <footer class="osu-layout__section osu-layout__section--landing-footer">
+        <div class="osu-layout__row osu-layout__row--landing-sitemap landing-sitemap">
+            <div class="osu-layout__col-container osu-layout__col-container--landing-sitemap">
+                <div class="osu-layout__col osu-layout__col--sm-6 osu-layout__col--lg-3">
+                    <ul class="landing-sitemap__list">
+                        <li class="landing-sitemap__item">
+                            <div class="landing-sitemap__header">General</div>
```

Comment:
Do you think I would be able to get away with trans("$section.title") for each of the pages? So for instance, if I want to link to ```Download osu!``` it would be in download.php under ```"title" => "Download osu!"```